### PR TITLE
ocrvs-3296 Removed internal calls to verifyToken from metrics, notifications, workflow, user-mgnt, search microservice

### PR DESCRIPTION
**Pull request by sadmananik**

### CODE REVIEW

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]
The constants need to be removed from the docker compose files as well

**Review by Zangetsu101 — APPROVED** [review]

**Comment by Zangetsu101** [comment]
We need to verify if the calls to verifyFunc have been reduced properly with kibana in staging

File: packages/notification/src/server.ts [hunk 1]
```diff
@@ -45,8 +42,9 @@ export async function createServer() {
       issuer: 'opencrvs:auth-service',
       audience: 'opencrvs:notification-user'
     },
-    validate: (payload: any, request: Hapi.Request) =>
-      validateFunc(payload, request, CHECK_INVALID_TOKEN, AUTH_URL)
+    validate: () => ({
+      isValid: true
+    })
```
[the remark this answers is not in the document]
Comment (sadmananik) [reply packages/notification/src/server.ts:47]: @euanmillar I have tried it first but found out this validate function is required as hapi js `server.auth.strategy` function parameter

File: docker-compose.yml [hunk 2]
```diff
@@ -145,10 +137,8 @@ services:
       dockerfile: ./packages/notification/Dockerfile
     restart: unless-stopped
     environment:
```
Comment (euanmillar) [inline docker-compose.yml:139]: As webhooks is externally facing, I think that webhooks still needs this

**Review by euanmillar — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (Zangetsu101) [reply docker-compose.yml:139]: @euanmillar You must have mixed the service names. As I don't see any changes in `webhooks`. Rather these are the changes for `notification`

Comment (euanmillar) [reply docker-compose.yml:139]: Ah yes, sorry "Git files changed" makes it look like these belong in webhooks.  My bad

**Pull request merged**